### PR TITLE
feat: top posts to most commented

### PR DESCRIPTION
**Pull request by MoMentalochka**
### Reopened pull after unwanted pull #582 deletion

### CODE REVIEW

**Comment by MoMentalochka** [comment]
Now in the side menu there are the Best and Discussed feeds, loading with a pagination step of 60 blocks and a maximum depth limit per month (43200 blocks)

**Review by maxgithubprofile — APPROVED** [review]

File: components/comments/index.js [hunk 1]
```diff
@@ -2767,7 +2785,7 @@ var comments = (function(){
 						alredyCommented = true;
 					}
 				}
-					
+
```
Comment (shpingalet007) [inline components/comments/index.js:2788]: Redundant linting

**Review by shpingalet007 — CHANGES_REQUESTED** [review]
### Redundant linting in these files
- components/lenta/index.js
- js/satolist.js

### And other issues...

**Review by maxgithubprofile — APPROVED** [review]

**Review by shpingalet007 — APPROVED** [review]
Quality checked

**Pull request merged**